packages/core/app-layout/docs/sidebar.md

```diff
@@ -223,6 +227,12 @@ Utilize the `header` slot to inject your application's logo into the top of the
 
 Utilize the `top` slot to inject additional UI into the top of the sidebar, below the `header` slot, above the `topItems`.
 
+#### `footer`
+
+Utilize the `footer` slot to inject content into the fixed area at the bottom of the sidebar, below the `bottomItems`.
+
+If `footer` slot content is provided, it will override the `profileItems` and `profileName`, preventing the sidebar bottom profile menu and popup menu from appearing.
```

Comment:
Question: If `footer` slot content is NOT provided, how long do we plan to continue to display `profileItems` and `profileName` in the bottom menu, by default? I mean, is this expected to change upon adding `profileItems` and `profileName` to the top nav Account Dropdown?